capture.py

```diff
@@ -798,7 +801,12 @@ def _get_screen_size():
     if _in_standalone():
         return [0, 0]
 
-    rect = QtWidgets.QDesktopWidget().screenGeometry(-1)
+    try:
+        rect = QtWidgets.QDesktopWidget().screenGeometry(-1)
+    except AttributeError:
+        # in Qt6 it is a different call
+        rect = QtWidgets.QApplication.primaryScreen().availableGeometry()
```

Comment:
Should this actually be 'reversed' if Qt6 is the new 'standard' moving forward? Like, should we default to the new way - and only fall back to the 'old' way otherwise?